src/Sylius/Bundle/PromotionBundle/Validator/CatalogPromotionActionValidator.php

```diff
@@ -34,12 +34,19 @@ public function validate($value, Constraint $constraint): void
         }
 
         $configuration = $value->getConfiguration();
+
```

Comment:
Unnecessary line IMO
```suggestion
```

Reply:
TBH for me, this space make this code more readable because this `$configuration` is used in 3 `if` statements below